### PR TITLE
Preparing release 1.15.0

### PR DESCRIPTION
**Pull request by guo0693**
Signed-off-by: Jun Guo <guo0693@gmail.com>

## Which problem is this PR solving?
- 

## Short description of the changes
-

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+* Small memory optimizations in badger write-path ([#1771](https://github.com/jaegertracing/jaeger/pull/1771), [@burmanm](https://github.com/burmanm))
+* Set an empty value when a default env var value is missing ([#1777](https://github.com/jaegertracing/jaeger/pull/1777), [@jpkrohling](https://github.com/jpkrohling))
+* Decouple storage dependencies and bump Go to 1.13.x ([#1886](https://github.com/jaegertracing/jaeger/pull/1886), [@yurishkuro](https://github.com/yurishkuro))
+* Add Truncate and ReadOnly options for badger ([#1842](https://github.com/jaegertracing/jaeger/pull/1842), [@burmanm](https://github.com/burmanm))
+* Update gopkg.in/yaml.v2 dependency to v2.2.4 ([#1865](https://github.com/jaegertracing/jaeger/pull/1865), [@objectiser](https://github.com/objectiser))
+* Upgrade jaeger-client 2.19 and jaeger-lib 2.2 and prom client 1.x ([#1810](https://github.com/jaegertracing/jaeger/pull/1810), [@yurishkuro](https://github.com/yurishkuro))
+* Unpin grpc version and use serviceConfig to set the load balancer  ([#1786](https://github.com/jaegertracing/jaeger/pull/1786), [@]())
```
Comment (yurishkuro) [inline CHANGELOG.md]: missing author?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
+* Propagate the bearer token to the gRPC plugin server ([#1822](https://github.com/jaegertracing/jaeger/pull/1822), [@radekg](https://github.com/radekg))
```
Comment (yurishkuro) [inline CHANGELOG.md]: I'd classify this as new feature

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,18 +1,34 @@
...
 ##### New Features
 
+* Support TLS for Kafka ([#1414](https://github.com/jaegertracing/jaeger/pull/1414), [@MichaHoffmann](https://github.com/MichaHoffmann))
+
 ##### Bug fixes, Minor Improvements
+* Use correct context on ES search methods ([#1850](https://github.com/jaegertracing/jaeger/pull/1850), [@rubenvp8510](https://github.com/rubenvp8510))
+* Handling of expected error codes coming from grpc storage plugins #1741 ([#1814](https://github.com/jaegertracing/jaeger/pull/1814), [@chandresh-pancholi](https://github.com/chandresh-pancholi))
+* Fix ordering of indexScanKeys after TraceID parsing ([#1809](https://github.com/jaegertracing/jaeger/pull/1809), [@burmanm](https://github.com/burmanm))
+* Fix flaky ES span reader test due to date differences ([#1847](https://github.com/jaegertracing/jaeger/pull/1847), [@pavolloffay](https://github.com/pavolloffay))
+* Add ack and compression parameters for Kafka #1359 ([#1712](https://github.com/jaegertracing/jaeger/pull/1712), [@chandresh-pancholi](https://github.com/chandresh-pancholi))
```
Comment (yurishkuro) [inline CHANGELOG.md]: I'd classify this as new feature

File: CHANGELOG.md [hunk 4]
```diff
@@ -85,6 +101,8 @@ Changes by Version
 
 ##### New Features
 
+* Support TLS for Kafka ([#1414](https://github.com/jaegertracing/jaeger/pull/1414), [@MichaHoffmann](https://github.com/MichaHoffmann))
+
```
Comment (yurishkuro) [inline CHANGELOG.md]: Was it released in 1.13.1 ?   You have it listed twice.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,18 +1,34 @@
 Changes by Version
 ==================
 
-1.15.0 (unreleased)
+1.15.0 (2019-11-06)
 ------------------
-
 #### Backend Changes
 
 ##### Breaking Changes
```
Comment (yurishkuro) [inline CHANGELOG.md]: weren't there some changes the the CLI flags? We usually list them (e.g. some flags may have been deprecated and replaced). You could dump `help` for each binary and do a diff (we may want to write & commit a script for that).

Comment (guo0693) [reply CHANGELOG.md]: There was a commit to add firehose flag for tracegen, do we want to add it to the changelog?

File: CHANGELOG.md (hunk 1 above)
Comment (guo0693) [reply CHANGELOG.md]: Hmm, I think make changelog somehow messed up with Jude's PR, was he removed from Uber git org? Going to manually add that

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,18 +1,56 @@
...
+* Fix flaky ES span reader test due to date differences ([#1847](https://github.com/jaegertracing/jaeger/pull/1847), [@pavolloffay](https://github.com/pavolloffay))
+* Small memory optimizations in badger write-path ([#1771](https://github.com/jaegertracing/jaeger/pull/1771), [@burmanm](https://github.com/burmanm))
+* Set an empty value when a default env var value is missing ([#1777](https://github.com/jaegertracing/jaeger/pull/1777), [@jpkrohling](https://github.com/jpkrohling))
+* Decouple storage dependencies and bump Go to 1.13.x ([#1886](https://github.com/jaegertracing/jaeger/pull/1886), [@yurishkuro](https://github.com/yurishkuro))
+* Update gopkg.in/yaml.v2 dependency to v2.2.4 ([#1865](https://github.com/jaegertracing/jaeger/pull/1865), [@objectiser](https://github.com/objectiser))
+* Upgrade jaeger-client 2.19 and jaeger-lib 2.2 and prom client 1.x ([#1810](https://github.com/jaegertracing/jaeger/pull/1810), [@yurishkuro](https://github.com/yurishkuro))
+* Unpin grpc version and use serviceConfig to set the load balancer  ([#1786](https://github.com/jaegertracing/jaeger/pull/1786), [@guanw](https://github.com/guanw))
 
 #### UI Changes
```
Comment (yurishkuro) [inline CHANGELOG.md]: please bump the ui submodule to the latest UI release and include the changes from there

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,18 +1,56 @@
 Changes by Version
 ==================
 
-1.15.0 (unreleased)
+1.15.0 (2019-11-07)
```
Comment (pavolloffay) [inline CHANGELOG.md]: Add a new section 1.16.0 (unreleased)

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,18 +1,56 @@
...
 #### Backend Changes
 
 ##### Breaking Changes
 
-* The default value for the Ingester's flag `ingester.deadlockInterval` has been changed to `0`. With the new default, the ingester won't `panic` if there are no messages for the last minute. To restore the previous behavior, set the flag's value to `1m`.
+* The default value for the Ingester's flag `ingester.deadlockInterval` has been changed to `0` ([#1868](https://github.com/jaegertracing/jaeger/pull/1868), [@jpkrohling](https://github.com/jpkrohling))
+
+  With the new default, the ingester won't `panic` if there are no messages for the last minute. To restore the previous behavior, set the flag's value to `1m`.
 
+* CLI flags have below changes:
```
Comment (pavolloffay) [inline CHANGELOG.md]: -1 on documenting added flags.

If some flags changed please list old deprecated flags and mention if they are working or they were removed.
Could you please add a link to PR which changed the flags.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,18 +1,56 @@
...
+* Add ack and compression parameters for Kafka #1359 ([#1712](https://github.com/jaegertracing/jaeger/pull/1712), [@chandresh-pancholi](https://github.com/chandresh-pancholi))
+* Propagate the bearer token to the gRPC plugin server ([#1822](https://github.com/jaegertracing/jaeger/pull/1822), [@radekg](https://github.com/radekg))
+* Add Truncate and ReadOnly options for badger ([#1842](https://github.com/jaegertracing/jaeger/pull/1842), [@burmanm](https://github.com/burmanm))
+
 ##### Bug fixes, Minor Improvements
+* Use correct context on ES search methods ([#1850](https://github.com/jaegertracing/jaeger/pull/1850), [@rubenvp8510](https://github.com/rubenvp8510))
+* Handling of expected error codes coming from grpc storage plugins #1741 ([#1814](https://github.com/jaegertracing/jaeger/pull/1814), [@chandresh-pancholi](https://github.com/chandresh-pancholi))
+* Fix ordering of indexScanKeys after TraceID parsing ([#1809](https://github.com/jaegertracing/jaeger/pull/1809), [@burmanm](https://github.com/burmanm))
+* Fix flaky ES span reader test due to date differences ([#1847](https://github.com/jaegertracing/jaeger/pull/1847), [@pavolloffay](https://github.com/pavolloffay))
```
Comment (pavolloffay) [inline CHANGELOG.md]: no need to be here

**Pull request merged**